.github/workflows/build.yml

```diff
@@ -2,6 +2,14 @@ name: Build
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+    paths:
```

Comment:
is there no way we can do an ignore paths instead?

Reply:
We can, this is how I had it before but we end up with a similar situation needing to ignore some individual files and paths i.e. README.md, deploy.sh to name just a couple. I can reverse it if you want?

Reply:
can we do something like
- '**'
- '!infrastructure/parameter/**'